doc/userguide/rules/dhcp-keywords.rst

```diff
@@ -6,6 +6,8 @@ dhcp.leasetime
 
 DHCP lease time (integer).
 
+dhcp.leasetime uses an :ref:`unsigned 64-bits integer <rules-integer-keywords>`.
```

Comment:
nit: Use `64-bit` instead of `64-bit` (similarly for other types: 8-bit, 16-bit, ...)

Reply:
> Use 64-bit instead of 64-bit

I do not see the difference.
Do you mean instead of `64-bits` with an s ?
(is it not a plural ? for my English knowledge)